code/modules/admin/verbs/debug.dm

```diff
@@ -692,6 +694,7 @@ GLOBAL_PROTECT(AdminProcCallSpamPrevention)
 			var/datum/outfit/O = path
 			if(initial(O.can_be_admin_equipped))
 				job_outfits[initial(O.name)] = path
+		job_outfits = sortTim(job_outfits, cmp=/proc/cmp_text_asc)
```

Comment:
Same here as above. Could be made static. But don't have to in my opinion seeing it is out of scope of the PR